app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -32,6 +36,11 @@
     padding-bottom: 14px;
   }
 
+  .govuk-breadcrumbs__list-item:before {
+    margin: initial;
```

Comment:
`initial` isn't supported by IE, so the breadcrumb looks like the below. I think the same effect could be achieved using `margin: 0`?

![Screenshot 2020-02-17 at 09 02 04](https://user-images.githubusercontent.com/861310/74638655-3a9a6d80-5164-11ea-8898-df05c642b137.png)

Reply:
good spot, have implemented.